terraform/modules/matrix-service/infra/redshift.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_redshift_cluster" "default" {
+  cluster_identifier = "dcp-matrix-service-cluster-${var.deployment_stage}"
+  database_name      = "${var.deployment_stage}"
```

Comment:
please namespace the database_name with matrix-service as well

Reply:
Will be named matrix_service_*, does not allow '-'

Reply:
sounds good